app/pages/project/instances/instance/InstancePage.tsx

```diff
@@ -54,6 +62,14 @@ export function InstancePage() {
     query: { project: instanceSelector.project },
   })
 
+  const { data: nicList } = usePrefetchedApiQuery('instanceNetworkInterfaceList', {
+    query: {
+      project: instanceSelector.project,
+      instance: instanceSelector.instance,
+      limit: 10,
```

Comment:
a stupid thing, but I think there is a max of 8 results here anyway. you can leave off the `limit` and rely on the API default of 100